Logger.cpp

```diff
@@ -28,7 +28,7 @@ void Logger::logResults() {
         << "\n8. Number of private data access: " << numPrivateDataAccess
         << "\n9. Number of public data access: " << numPublicDataAccess
         << "\n10: Number of unique data access: " << numUniqueAddresses
-        << std::endl << std::endl;
+        << "\n" << std::endl;
```

Comment:
Could you explain this change please? Why is `std::endl` unnecessary?

Reply:
`std::endl` flushes the I/O, which is slow, so using `"\n"` instead of `std::endl` allows us to buffer the I/O until we really need to print it.